stylesheet/sytle.css

```diff
@@ -1,9 +1,9 @@
-                #index-body {
-                    background-image: url("../images/main-image-2.gif");
```

Comment:
###[OPTIONAL]
You're adding an image to the body, this is not ideal since you might want to add more content in your site , however, this practice will restrict that activity.
Instead, you can add this background in a separate HTML section.